setup.py

```diff
@@ -1,15 +1,35 @@
 import setuptools
 from os import path
-from distutils.command.build_ext import build_ext as DistUtilsBuildExt
-from setuptools import setup, find_packages
+from setuptools import setup, find_packages, Extension
```

Comment:
Once this gets merged in, we'll need to make a new release and double check that all of this was formatted formatted appropriately. I don't know if there's a better way to check that all of this is correct

Reply:
For sure.  We should be good, since I followed the cython guides plus some other example cython projects to see what they do, but a double check on that never hurts.

Also, if we want stuff like autodoc strings in .pyx files, there's a little more overhead with a 'declaration' file and configuring some things in `ark/utils/__init__.py` which can make that happen, but that's a little extra imo